po/git-novice.ja.po

```diff
@@ -5409,14 +5409,14 @@ msgid ""
 "reference#protocol) from\n"
 "SSH to HTTPS."
 msgstr ""
-"Click on the 'HTTPS' link to change the [protocol]({{ page.root }}/"
-"reference#protocol) from\n"
-"SSH to HTTPS."
```

Comment:
(we'll get to this later, but it's interesting to note that the current lesson version now does the opposite! SSH instead of HTTPS)